user/btop/template.py

```diff
@@ -0,0 +1,20 @@
+pkgname = "btop"
+pkgver = "1.3.2"
+pkgrel = 0
+build_style = "makefile"
+make_cmd = "gmake"
+hostmakedepends = ["gmake"]
+pkgdesc = "TUI monitor of system resources"
+maintainer = "mia <mia@mia.jetzt>"
+license = "Apache-2.0"
+url = "https://github.com/aristocratos/btop"
+source = f"{url}/archive/v{pkgver}/{pkgname}-{pkgver}.tar.gz"
+sha256 = "331d18488b1dc7f06cfa12cff909230816a24c57790ba3e8224b117e3f0ae03e"
+hardening = ["cfi", "vis"]
+make_build_args = ["OPTFLAGS="]
+make_use_env = True
```

Comment:
these go near the top (see ci fail)

Reply:
there's also some other thing to clean up in the makefile but i'll do that for you after

Reply:
> these go near the top (see ci fail)  

(after rebase, since it was broken on master)  

you can also run black+flake8 locally, same thing

Reply:
> there's also some other thing to clean up in the makefile but i'll do that for you after  

or well, you could use cmake instead since that looks better, for which it's just  

```
build_style = "cmake"
hostmakedepends = ["cmake", "ninja"]
```

Reply:
+ `configure_args = ["-DBTOP_LTO=OFF", "-DBTOP_FORTIFY=OFF"]` and that's about it for cmake instead

Reply:
full template:  

```
pkgname = "btop"
pkgver = "1.3.2"
pkgrel = 0
build_style = "cmake"
configure_args = ["-DBTOP_LTO=OFF", "-DBTOP_FORTIFY=OFF"]
hostmakedepends = ["cmake", "ninja"]
pkgdesc = "TUI monitor of system resources"
maintainer = "mia <mia@mia.jetzt>"
license = "Apache-2.0"
url = "https://github.com/aristocratos/btop"
source = f"{url}/archive/v{pkgver}/{pkgname}-{pkgver}.tar.gz"
sha256 = "331d18488b1dc7f06cfa12cff909230816a24c57790ba3e8224b117e3f0ae03e"
hardening = ["cfi", "vis"]


# no tests, just make sure binary runs
def do_check(self):
    self.do("./build/btop", "--version")
```

note also the self.do, for cwd it's fine to use `./` instead of the longer form

Reply:
so many things!
it have put them into the git machine, thank you

Reply:
hah yeah, i talk a lot sorry